news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 15 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-04-19 16:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 8th to Sunday 14th of April 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+The tests and debug phase of the 1.7.6 release has started: 134 issues have been created on the week after the code freeze. Some developers are already testing it on their infrastructure, opening issues, and submitting pull requests, and that is AWESOME!!!
+
+Please continue like that, thank you very much for your feedback.
+
+Oh, and this week, I went to the first edition of the PSconnect event in Lyon, France, with a few people of the PrestaShop company: business managers, product managers, our CEO and our CTO. That was a very interesting moment to discuss and collect feedback about the roadmap. Merchants, partners, agencies and freelancers confirmed that the project's roadmap is on good tracks. There is a very good feedback about 1.7.5, with many projects in production and scheduled migrations.
```

Comment:
to discuss and collect feedback ==> feedbacks

Reply:
feedback is usually without s.